### PR TITLE
3/context menu btn

### PR DESCRIPTION
**Pull request by abea**
Resolves https://apostrophecms.atlassian.net/browse/A30U-806, "Style the Undo/Redo Buttons," and finishes https://apostrophecms.atlassian.net/browse/A30U-795,  "Style the context bar"

The undo/redo icons aren't exactly the same as in the design, but they are what's included in Material icons. How do we feel about that?

Also resolves https://apostrophecms.atlassian.net/browse/A30U-735, "Z-index conflict remaining on area buttons"

### CODE REVIEW

File: modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue [hunk 1]
```diff
@@ -54,7 +54,22 @@
...
+            icon="undo-icon" :icon-only="true"
+            @click="undo"
+          />
+          <AposButton
+            :disabled="undone.length === 0"
+            type="default" label="Redo"
+            class="apos-admin-bar__context-button"
+            icon="redo-icon" :icon-only="true"
+            @click="redo"
+          />
```
Comment (stuartromanek) [inline modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue:71]: THese need tooltips

Comment (stuartromanek) [reply modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue:71]: Also, use the undo/redo variants

image

**Review by stuartromanek — CHANGES_REQUESTED** [review]

File: modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue (hunk 1 above)
Comment (stuartromanek) [reply modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue:71]: Also also, these buttons should use the `outline` and `no-motion` modifiers

**Review by stuartromanek — CHANGES_REQUESTED** [review]
forgot one

**Review by localghost443 — APPROVED** [review]

**Comment by abea** [comment]
We're back to think looking junky due to the autosave merge. I'm ignoring that for now since we'll need design direction for that.

**Review by stuartromanek — APPROVED** [review]

**Pull request merged**